cfme/infrastructure/virtual_machines.py

```diff
@@ -538,6 +545,12 @@ def mem_size_mb(self):
         return self.mem_size * 1024 if self.mem_size_unit == 'GB' else self.mem_size
 
 
+class NetworkAdapter(namedtuple("NetworkAdapter", ["name", "vlan"])):
+    # just compare name not vlan
+    def __eq__(self, other):
+        return self.name == other.name
```

Comment:
**Question:** Why do you not compare vlan?

Reply:
default will compare object instance... but after fetch data from db we won't have `vlan` type; so need this arrangement.